ComfyUI/comfy_api_nodes/nodes_runway_text2img.py

```
@@ -0,0 +1,269 @@
+       
+        # Validate environment and inputs
+        self.validate_environment()
+        validate_string(prompt, min_length=1, field_name="prompt")
+        
+        # Prepare reference images if provided
+        reference_images = None
+        if reference_image is not None:
+            # Validate reference image dimensions
+            if len(reference_image.shape) != 4 or reference_image.shape[1] not in {1, 3, 4}:
+                raise ValueError()
+                
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)